### PR TITLE
Create a "Supported Cluster Modifications" topic

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/894

We have "Reach out to Astronomer Support" statements scattered throughout the docs, but no central place where we maintain information about what modifications are supported and can only be completed by Astronomer support. @kwcanuck Especially interested in your thoughts on this sort of reference topic that doesn't provide any new info but simply compiles similar info from many disparate docs.

### CODE REVIEW

File: astro/kubernetespodoperator.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ To complete this setup, you need:
 To run Docker images from a private registry on Astro, you first need to create a Kubernetes secret that contains credentials to your registry. Astronomer will then inject that secret into your Deployment's namespace, which will give your tasks access to Docker images within your registry. To do this, complete the following setup:
 
 1. Log in to your Docker registry and follow the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#log-in-to-docker-hub) to produce a `/.docker/config.json` file.
-2. Reach out to [Astronomer support](https://support.astronomer.io) and include the namespace of the Deployment you want to use the KubernetesPodOperator with. A Deployment's namespace can be found in the Deployment view of the Astronomer UI.
+2. Reach out to [Astronomer support](https://support.astronomer.io) and provide the namespace of the Deployment you want to use the KubernetesPodOperator with. A Deployment's namespace can be found in the Deployment view of the Astronomer UI.
```
Comment (kwcanuck) [inline astro/kubernetespodoperator.md:114]:
```suggestion
2. Contact [Astronomer support](https://support.astronomer.io) and provide the namespace of the Deployment you want to use the KubernetesPodOperator with. A Deployment's namespace can be found in the Deployment view of the Astronomer UI.
```
Should instructions for locating the Deployment namespace be provided? For example:

1.  Log in to your Docker registry and create a `/.docker/config.json` file. See [Log in to Docker Hub](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#log-in-to-docker-hub). 

2. In the Cloud UI, select a Workspace and then select a Deployment.

3. Copy the value in the **NAMESPACE** field. 

4. Submit a support request and provide the the namespace value for the Deployment you want to use with the KubernetesPodOperator. See [Submit a support request](astro-support.md)

File: astro/kubernetespodoperator.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ To complete this setup, you need:
...
 
 From here, Astronomer Support will give you instructions on how to securely send the output of your `/.docker/config.json` file. Do not send this file via email, as it contains sensitive credentials to your registry.
```
Comment (kwcanuck) [inline astro/kubernetespodoperator.md:116]:
```suggestion
Astronomer Support will give you instructions on how to securely send the output of your `/.docker/config.json` file. Do not send this file by email, as it contains sensitive credentials to your registry.
```

File: astro/modify-cluster.md [hunk 3]
```diff
@@ -27,3 +27,13 @@ To modify an existing cluster in your Organization, first verify that the change
 Once our team validates that the cluster configuration you requested is supported, we will let you know as soon as we are able to perform the change.
```
Comment (kwcanuck) [inline astro/modify-cluster.md:27]:
```suggestion
 You are notified when Astronomer support validates that the cluster configuration you requested is supported and can be implemented.
```

**Comment by kwcanuck** [comment]
@jwitz in response to your comments:

File: astro/modify-cluster.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ description: Request changes to an existing Astro cluster.
 
 Unless otherwise specified, new Clusters on Astro are created with a set of default configurations. Depending on your use case, you may decide that you want to modify an existing Cluster to run a different configuration.
 
-For example, if you have a new set of DAGs that require significantly more CPU and Memory than your existing workloads, you may be interested in modifying a cluster on AWS to run `m5.8xlarge` nodes instead of `m5.4xlarge` nodes. You might also want to modify a cluster's maximum node count from the default of 20 to better fit your expected workload.
+For example, if you have a new set of DAGs that require significantly more CPU and Memory than your existing workloads, you might be interested in modifying a cluster on AWS to run `m5.8xlarge` nodes instead of `m5.4xlarge` nodes. You might also want to modify a cluster's maximum node count from the default of 20 to better fit your expected workload.
 
 ## Prerequisites
```
Comment (paolaperaza) [inline astro/modify-cluster.md:12]: Looking at the header hierarchy, I actually wonder if it'd make sense to do:

## Review supported cluster modifications
- This is where you'd have the list you added
- Link to AWS + GCP resource reference

## Reach out to Astronomer

## Confirm your cluster change
- Let's actually replace the "confirm with Astronomer" piece to just have them look in the Cloud UI? We can have a statement that says Astronomer will confirm, but we should add that the user can also just go to Cloud UI:

image

File: astro/modify-cluster.md [hunk 5]
```diff
@@ -18,12 +18,24 @@ To complete this setup, you need to have:
...
 
-## Step 1: Submit a Request to Astronomer
+## Supported cluster modifications
+
+Some cluster and Deployment-level modifications can be completed only by Astronomer support. These include:
+
+- [Creating a new cluster](create-cluster.md).
+- Updating a cluster's worker instance type.
+- Updating the maximum node count of an existing cluster.
+- [Creating a VPC connection](connect-external-services.md#vpc-peering) between a cluster and a target VPC.
```
[the remark this answers is not in the document]
Comment (jwitz) [reply astro/modify-cluster.md:28]: We have docs for it now approved by Deepak: https://github.com/astronomer/docs/pull/882#event-7015750790

File: astro/modify-cluster.md [hunk 6]
```diff
@@ -18,12 +18,25 @@ To complete this setup, you need to have:
...
+## Supported cluster modifications
+
+Some cluster and Deployment-level modifications can be completed only by Astronomer support. These include:
+
+- [Creating a new cluster](create-cluster.md).
+- Deleting a cluster.
+- Updating a cluster's worker instance type. See cloud resource references ([AWS](resource-reference-aws.md#deployment-worker-size-limits), [GCP](resource-reference-gcp.md#deployment-worker-size-limits)).
+- Updating the maximum node count of an existing cluster.
+- [Creating a VPC connection](connect-external-services.md#vpc-peering) between a cluster and a target VPC.
+- Running images from a private registry with the [KubernetesPodOperator](kubernetespodoperator#run-images-from-a-private-registry).
```
Comment (paolaperaza) [inline astro/modify-cluster.md:30]:
```suggestion
- Run Docker images from a private registry with the [KubernetesPodOperator](kubernetespodoperator#run-images-from-a-private-registry).
```

File: astro/modify-cluster.md [hunk 7]
```diff
@@ -18,12 +18,25 @@ To complete this setup, you need to have:
 
 If you don't have a cluster on Astro, follow the instructions to [Install Astro on AWS](install-aws.md) or [GCP](install-gcp.md). If you have an existing cluster and are interested in creating additional clusters, read [Create a cluster](create-cluster.md).
 
-## Step 1: Submit a Request to Astronomer
+## Supported cluster modifications
+
+Some cluster and Deployment-level modifications can be completed only by Astronomer support. These include:
```
Comment (paolaperaza) [inline astro/modify-cluster.md:23]:
```suggestion
Some cluster and Deployment-level modifications require Astronomer support and cannot be completed with the Cloud UI or CLI. These include requests to:
```

File: astro/modify-cluster.md [hunk 8]
```diff
@@ -18,12 +18,25 @@ To complete this setup, you need to have:
...
+
+- [Creating a new cluster](create-cluster.md).
```
Comment (paolaperaza) [inline astro/modify-cluster.md:25]:
```suggestion
- [Create a new cluster](create-cluster.md).
```

File: astro/modify-cluster.md [hunk 9]
```diff
@@ -18,12 +18,25 @@ To complete this setup, you need to have:
...
+- Deleting a cluster.
```
Comment (paolaperaza) [inline astro/modify-cluster.md:26]:
```suggestion
- Delete an existing cluster.
```

File: astro/modify-cluster.md [hunk 10]
```diff
@@ -18,12 +18,25 @@ To complete this setup, you need to have:
...
+- Updating a cluster's worker instance type. See cloud resource references ([AWS](resource-reference-aws.md#deployment-worker-size-limits), [GCP](resource-reference-gcp.md#deployment-worker-size-limits)).
```
Comment (paolaperaza) [inline astro/modify-cluster.md:27]:
```suggestion
- Update a cluster's worker instance type. See cloud resource references ([AWS](resource-reference-aws.md#deployment-worker-size-limits), [GCP](resource-reference-gcp.md#deployment-worker-size-limits)).
```

File: astro/modify-cluster.md [hunk 11]
```diff
@@ -18,12 +18,25 @@ To complete this setup, you need to have:
...
+- Updating the maximum node count of an existing cluster.
```
Comment (paolaperaza) [inline astro/modify-cluster.md:28]:
```suggestion
- Update the maximum node count of an existing cluster.
```

File: astro/modify-cluster.md [hunk 12]
```diff
@@ -18,12 +18,24 @@ To complete this setup, you need to have:
...
 
 To modify an existing cluster in your Organization, first verify that the change you want to make is supported by reading the resource reference documentation for either [AWS](resource-reference-aws.md) or [GCP](resource-reference-gcp.md). Then, reach out to [Astronomer support](https://support.astronomer.io).
 
-## Step 2: Confirm with Astronomer
+## Step 2: Confirm the modification
+
+If the modification you requested is supported, Astronomer will notify you as soon as it's possible to complete the modification.
 
-Once our team validates that the cluster configuration you requested is supported, we will let you know as soon as we are able to perform the change.
+Modifications to an existing cluster might take a few minutes to complete, but you can expect no downtime during the process. Astro is built to ensure a graceful rollover, which means that the Airflow and Cloud UIs will continue to be available and your Airflow tasks will not be affected.
```
[the remark this answers is not in the document]
Comment (paolaperaza) [reply astro/modify-cluster.md:40]: Yes -- tweaking wording here per @collinmcnulty suggestion

File: astro/modify-cluster.md [hunk 13]
```diff
@@ -18,12 +18,25 @@ To complete this setup, you need to have:
...
```
Comment (paolaperaza) [inline astro/modify-cluster.md:40]:
```suggestion
Most modifications to an existing cluster take only a few minutes to complete and do not incur downtime. In these cases, the Airflow UI and Cloud UI will continue to be available and your Airflow tasks are not interrupted.

For modifications that do incur downtime, such as changing your cluster's node instance type, Astronomer support will inform you of the expected impact and ask that you confirm if you want to proceed.
```

**Review by paolaperaza — APPROVED** [review]
A few thoughts here @jwitz but will let you polish off! Included copy that @collinmcnulty suggested would be appropriate for this doc re: downtime for cluster changes

**Pull request merged**